src/components/image/image.stories.mdx

```diff
@@ -0,0 +1,134 @@
+import { Meta, Story, Preview } from "@storybook/addon-docs/blocks";
+import { ArgsTable } from "@storybook/components";
+
+import Image from "./image.component";
+import Box from "../box";
+import Typography from "../typography";
+import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
+import useMediaQuery from "../../hooks/useMediaQuery/index.js"
+
+<Meta title="Design System/Image" />
+
+# Image
+Basic component that allows either the rendering of an `img` element or a `div` with a `background-image`. This component 
+also gives access to a number of styling options such as `margin`, and `layout`.
+
+## Contents
+- [Quick Start](#quick-start)
+- [Examples](#examples)
+- [Props](#props)
+
+## Quick Start
+Import the `Image` component to start using
+
+```javascript
+import Image from "carbon-react/lib/components/image";
+```
+
+## Examples
+
+### Default
+By default the component will render as a `div` element and use the `backgroundImage` to set the the `background-image` 
+css attribute.
+
+<Preview>
+  <Story name="default" parameters={{ info: { disable: true } }}>
+    <Image m={3} height="700px" backgroundImage={`url("https://carbon.sage.com/static/media/point.6992f190.svg")`}>
+      <Box height="700px" display="flex" justifyContent="center">
+        <Box display="flex" flexDirection="column" justifyContent="center">
```

Comment:
Is this second `Box` just to vertically align centre? If so just put a `alignItems="center"` on the first `Box` instead

Reply:
Yep good shout, I had a more complex example previously but scrapped it